CMakeLists.txt

```diff
@@ -98,6 +98,8 @@ if(SHARED_MODEL_DISABLE_COMPATIBILITY)
     add_definitions(-DDISABLE_BACKWARD)
 endif()
 
+SET(IROHA_ROOT_PROJECT ON)
```

Comment:
`set(IROHA_ROOT_PROJECT ON CACHE BOOL "Description")`

or 

`option(IROHA_ROOT_PROJECT "Description" ON)`

currently you have no way to change this var from terminal without changing CMakeLists.txt

Reply:
But I don't want to change it from terminal.

Reply:
Otherwise you do not have any control is n this variable, it will be always ON

Reply:
Yes, it is correct.